fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java

```diff
@@ -1523,6 +1523,11 @@ private void updateCatalogAfterCommitted(TransactionState transactionState, Data
         for (TableCommitInfo tableCommitInfo : transactionState.getIdToTableCommitInfos().values()) {
             long tableId = tableCommitInfo.getTableId();
             Table table = db.getTable(tableId);
+            if (table == null) {
```

Comment:
If the table was force dropped, isn't it possible to have `table==null` here as well?